classes/game_author.rb

```diff
@@ -0,0 +1,61 @@
+# This is item.rb it reps an item class
+module GameAuthor
+  def list_games
+    if @games.empty?
+      puts 'No games found.'
+    else
+      puts 'List of Games:'
+      @games.each_with_index do |game, index|
+        display_game(game, index)
+      end
+    end
+  end
+
+  def display_game(game, index)
+    id_text = "ID: #{game.id}"
+    publish_date_text = "Published Date: #{game.publish_date}"
+    multiplayer_text = "Multiplayer: #{game.multiplayer ? 'Yes' : 'No'}"
+    last_played_text = "Last Played: #{game.last_played_at}"
+
+    puts "#{index + 1}, #{id_text}, #{publish_date_text}, #{multiplayer_text}, #{last_played_text}"
+  end
+
+  def list_authors
+    if @authors.empty?
+      puts 'No authors found.'
+    else
+      puts 'List of Authors:'
+      @authors.each do |author|
+        puts "ID: #{author.id}, Name: #{author.full_name}"
+      end
+    end
```

Comment:
- [ ] Also, I realized when I checked the list of authors added, it throws a prompt saying `No author found` and I believe it should not work like that. when I added books, movies, and games, we added the publisher's data in the prompt, so I believe in this context publisher should serve as the author of the book, movie, and game so I shouldn't be getting `No authors found`. I believe this is also a bug in your project and I would like you to work on it so that we can get the list of the authors

Screenshot 👇 

![catalog3](https://github.com/2004-okari/CATALOG/assets/78634841/f16afdf9-29c5-4069-8ae9-c0397131fefc)